Asterix/Example_param_file.ini

```diff
@@ -55,7 +55,7 @@ onbench=True
     # if True, all the pupils will be created 10x larger than diam_pup_in_pix and 
     # then rebinned to diam_pup_in_pix at the exception of pupils that are read in 
     # a fit file, which are always rebinned from the sixe of the .fits
-    grey_pupils = False
+    grey_pupils = True
```

Comment:
Well. FQPM is in perfect coronagraph currently so contrast is always 0.
If we remove perfect coronagraph we go from 6.98 10-6 (without grey pup) to 6.14 10-6 (with grey pup) with the parameters as they are in param_file_tests.ini

I agree it would probably be good practice to test on the tesbed also.

Reply:
Oh right. Then it should be fine. Agreed with testing on the testbed, but we're not gonna change phase masks just for a software test and risk messing up our current alignment, so we'll manage as-is. Thanks for the clarification!